### DISCUSSION TITLE
how to run blocky on mipsle

### DISCUSSION THREAD

**Discussion by Tonho888** [0xERR0R/blocky#828] [Q&A]
maybe blocky will run on mipsle, and where can i download it, thank you for your attention

**Comment by kwitsch** [comment]
blocky is currently only compiled and tested on x64 & ARM.
Trying to run it on MIPS should be possible if your system has Golang 1.19 & Makefile support.
Compilation can be done through the Makefile with `make build`.

**Reply by Tonho888** [reply]
I'm not familiar with makefiles and builds, but my friend helped build them with commands GOOS=linux GOARCH=mipsle GOMIPS=softfloat CGO_ENABLED=0 go build -ldflags "-s -w" , and the results are like in the video, but is this true…

**Reply by kwitsch** [reply]
The parameters look fine to me, glad it works for you. :)

**Comment by Tonho888** [comment]
![2DA79DF1-6519-4D61-8BC5-1633A39374EB](https://user-images.githubusercontent.com/122684745/212845791-593db488-06d7-4114-b439-d677a03d0ddf.jpeg)

**Reply by Tonho888** [reply]
blocky runs on linksys ea8100 mipsle softload
